utilities/console.py

```
@@ -51,6 +51,7 @@ def __init__(
         self.cmd = self._generate_cmd()
         self.base_dir = get_data_collector_base_directory()
 
+    @retry(wait_timeout=TIMEOUT_5MIN, sleep=TIMEOUT_10SEC)
```

Flake8 (7.3.0)

[error] undefined name 'TIMEOUT_10SEC'

(F821)